examples/exporting_for_inference.ipynb

```diff
@@ -45,14 +30,38 @@
     "%pip install mosaicml"
    ]
   },
+  {
+   "cell_type": "markdown",
```

Comment:
I think starting with the method call itself is probably the simplest introduction, and then have a separate section on using the callback if you want to export periodically during training.

Reply:
Cool. I can move that part to the top. You mean the trainer.export_for_inference or our standalone api utils.inference.export_for_inference?

Reply:
Moved dedicated api export at the top with torchscript.